Makefile

```diff
@@ -14,34 +14,53 @@ INSTALL_DIR = $(INSTALL) -d
 INSTALL_PROGRAM = $(INSTALL) -m755
 INSTALL_DATA = $(INSTALL) -m644
 
+IS_ARCH_ARM := $(grep -q ARM /proc/cpuinfo; echo $$?)
+ifeq ($(IS_ARCH_ARM), 0)
```

Comment:
I still this this is not working.  The variable IS_ARCH_ARM does not get assigned 0 or 1 it seems to be a null so the check always ends up as x86.